src/controller/reducers/resource.js

```diff
@@ -0,0 +1,69 @@
+import {
+  RESOURCE_LOADING,
+  RESOURCE_SUCCESS,
+  SET_PARENT,
+  ADD_RESOURCE,
+  UPDATE_RESOURCE,
+  DELETE_RESOURCE,
+  RESOURCE_ERROR
+} from '../actions/resource';
+import produce from 'immer';
+
+export const initialState = {
+  resources: [],
+  recentResources: [],
+  loading: false,
+  error: null,
+  parent: {}
```

Comment:
Please explain to me what parent does when you read this comment. I'm not fully understanding it.

Reply:
I need think of a better way of doing this because right now it grabs this from the first resource. But if there are no resources, than this is empty and pointless which is not what I was going for.

So the parent here when it's not empty should hold the id and title of the parent of the resources array. When a user clicks on the topicDash the idea was that this should be `parent{ id:12345, title:Javascript }`. I was using this in the topicDash index.js to display the name of the topic. I also need it in addResourceForm to grab the ID and submit a new resource. Though, thinking about it I can grab it from route. 

If we take this out,  the questions is how can I grab the name of the parent so I can display on the topicDash. Do you have any ideas here? Or do you think this should stay in reducer?

Reply:
Removed from the controller. Will be fetching data from routes